extensions/data-plane-transfer/README.md

```diff
@@ -1,13 +1,9 @@
-# Data Plane Transfer extension
+# Data Plane Transfer
 
-This extension provides resources used to delegate data transfer to the Data Plane, or to use the Data Plane as a proxy for querying the data.
+This module provides extensions for performing data transfer through the Data Plane. Each extension is designed to
+support one type of data transfer.
 
-The setting parameters of this extension are listed below:
+## Extensions
 
-| Parameter name                                      | Description                                                                                                                        | Mandatory | Default value                          |
-|:----------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------|:----------|:---------------------------------------|
-| `edc.transfer.proxy.token.signer.privatekey.alias`  | Alias of private key used to sign token used to hit Data Plane public API                                                          | true      |                                        |
-| `edc.transfer.proxy.token.verifier.publickey.alias` | Alias of public key used to verify tokens hitting the Data Plane public API (public key must be in the Vault)                      | false     | private key alias suffixed with "-pub" |
-| `edc.transfer.proxy.token.validity.seconds`         | Validity of tokens generated for hitting Data Plane public API (in seconds)                                                        | false     | 600                                    | 
-| `edc.transfer.proxy.endpoint`                       | Public API endpoint of the Data Plane (TODO: this has to be dynamically generated by the Data Plane Selector) based on the request | true      |                                        |
-| `edc.transfer.client.selector.strategy`             | Selection strategy used by the client to determine to which Data Plane instance data transfer should be delegated                  | true      |                                        |
+- [Data Plane Transfer Client](data-plane-transfer-client) provides a client to delegate data transfer to the Data Plane.
```

Comment:
these links don't resolve, they should be put as `(./data-plane-transfer-client/<filename>.md)`

Reply:
nice catch!